engine/server/engine/enclave_manager/enclave_creator.go

```diff
@@ -85,20 +85,18 @@ func (creator *EnclaveCreator) CreateEnclave(
 
 	// only create log collector for backend as
 	shouldDeleteLogsCollector := true
-	if kurtosisBackendType == args.KurtosisBackendType_Docker {
-		// TODO the logs collector has a random private ip address in the enclave network that must be tracked
-		if _, err := creator.kurtosisBackend.CreateLogsCollectorForEnclave(setupCtx, enclaveUuid, defaultTcpLogsCollectorPortNum, defaultHttpLogsCollectorPortNum); err != nil {
-			return nil, stacktrace.Propagate(err, "An error occurred creating the logs collector with TCP port number '%v' and HTTP port number '%v'", defaultTcpLogsCollectorPortNum, defaultHttpLogsCollectorPortNum)
-		}
-		defer func() {
-			if shouldDeleteLogsCollector {
-				err = creator.kurtosisBackend.DestroyLogsCollectorForEnclave(teardownCtx, enclaveUuid)
-				if err != nil {
-					logrus.Errorf("Couldn't cleanup logs collector for enclave '%v' as the following error was thrown:\n%v", enclaveUuid, err)
-				}
-			}
-		}()
+	// TODO the logs collector has a random private ip address in the enclave network that must be tracked
+	if _, err := creator.kurtosisBackend.CreateLogsCollectorForEnclave(setupCtx, enclaveUuid, defaultHttpLogsCollectorPortNum, defaultTcpLogsCollectorPortNum); err != nil {
```

Comment:
now that kubernetes backend implements it, we can call this function at the engine level - at this point the engine should've already started the logs collector daemonset, so this will just serve as a check that its up and running for the k8s backend